centraldirectory/values.yaml

```diff
@@ -94,12 +99,13 @@ service:
         externalPort: 3000
 
 ingress:
-  enabled: false
+  enabled: true
   externalPath: /
   # Used to create an Ingress record.
   hosts:
-    api: central-directory.local
+    api: central-directory.oss.test
```

Comment:
this should be 'central-directory.local'.

Reply:
So we wanted to have a naming convention that is <service_name>.<customer>.<env> right. Cos we might be using the same Mojaloop helm chart, but the service names need to be different based on the deployment.